ui/client/datasets/FormFields/index.js

```diff
@@ -33,10 +33,17 @@ export const FormAwareTextField = withStyles((theme) => ({
     }
   },
 }))(({
-  classes, name, label, requiredFn, placeholder, inputProps={}, InputProps={}, required, ...props
+  classes, name, label, requiredFn, placeholder, inputProps={}, InputProps={}, onBlur, required, ...props
```

Comment:
Ended up not passing in an onBlur, but no harm for now. I might revert to simplify this component.